main.go

```diff
@@ -133,18 +134,19 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Initialize event recorder.
+	record.InitFromRecorder(mgr.GetEventRecorderFor("aws-controller"))
```

Comment:
```gcp-controller```

Reply:
whoops! Are you cool with fixing it as part of the other PR you have open?